### PR TITLE
Sscs 10954 local queue solution

### PR DESCRIPTION
**Pull request by ss-evoco**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by ss-evoco** [comment]
The reason for the failed build is CVEs caused by the lower version of spring. This will soon be fixed as part of SSCS-10358 (currently in test). Once its merged, I will rebase and then test locally again, alongside watching the jenkins output. In the mean time, please review the code as if it was building.

**Review by DGordonV1 — APPROVED** [review]

**Review by DGordonV1 — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/JmsMessagingServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+@RunWith(MockitoJUnitRunner.class)
+public class JmsMessagingServiceTest {
+
+    @Mock
+    private JmsTemplate jmsTemplate;
+
+    @Mock
+    private SessionAwareRequest message;
+
+    private JmsMessagingService jmsMessagingService;
```
[the remark this answers is not in the document]
Comment (ss-evoco) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/JmsMessagingServiceTest.java:23]: But the queueName isnt a mock.

File: src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/hearings/HearingMessagingServiceFactoryTest.java [hunk 2]
```diff
@@ -17,17 +19,28 @@ public class HearingMessagingServiceFactoryTest {
...
 
+    @Test
+    public void getMessagingService_HearingRouteListAssist_Jms()  {
+        HearingMessagingServiceFactory hearingMessagingServiceFactory = new HearingMessagingServiceFactory(null, null);
+        ReflectionTestUtils.setField(hearingMessagingServiceFactory, "jmsEnabled", true);
+
+        SessionAwareMessagingService messagingService = hearingMessagingServiceFactory
+            .getMessagingService(HearingRoute.LIST_ASSIST);
+
+        assertThat(messagingService.getClass()).isEqualTo(JmsMessagingService.class);
```
[the remark this answers is not in the document]
Comment (ss-evoco) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/hearings/HearingMessagingServiceFactoryTest.java:38]: It is assertj

File: src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/JmsMessagingServiceTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (TeaWithLucas) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/servicebus/JmsMessagingServiceTest.java:26]: Yeah your right nvm

**Review by TeaWithLucas — APPROVED** [review]

**Review by DGordonV1 — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]

**Review by nickhill111 — APPROVED** [review]

**Pull request merged**